src/components/button/button.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import './button.css'
+
+
+export default function Button(props: any): React.ReactElement {
```

Comment:
can be functional component as well `const Button = (props) => <button ... `

Reply:
is it any differences in this two approaches:

export function Button(props: any): React.ReactElement {

and 

export const Button = (props) =>